docs/cql/dml.rst

```diff
@@ -601,7 +601,7 @@ by the same node are unique. Timestamps assigned at different
 nodes are not guaranteed to be globally unique.
 With a steadily high write rate timestamp collision
 is not unlikely. If it happens, i.e. two ``INSERTS`` have the same
-timestamp, the lexicographically bigger value prevails:
+timestamp, a tie-breaking algorithm determines which if the inserts prevails (see :ref:`update ordering <update-ordering>`).
```

Comment:
This phrasing (also before your change, but also after) seems to suggest than one of the two INSERTS will prevail, while in reality, under some conditions, the result can be a weird mix of the two (with each column getting the maximum value from the two inserts).

Reply:
I agree that the text is quite vague but taking it simply, it does imply that the INSERT is considered in whole, rather than selecting which cell will prevail, individually.

We can add a note about that in the update-ordering section.

Reply:
Also, not that we can't guarantee that "insert atomicity" if neither of the inserts have a TTL.
In that case, the cells values are used for cell selection, independently, and the result could very well be a mixture of the inserts (since a. we compare the cells independently and not the whole row-mutation, and b. we don't keep a write time on non-expiring cells that could use to break the tie more elegantly)

Reply:
Exactly, which is why I'm saying that it's not a great idea to have wording that suggests we can always choose "which insert prevails".